src/org/daisy/dotify/api/formatter/BlockProperties.java

```diff
@@ -431,7 +433,18 @@ public Builder widows(int value) {
 			this.widows = value;
 			return this;
 		}
-		
+
+		/**
+		 * Sets the display when expression
+		 *
+		 * @param value     Expression to decide if the block should be shown
+		 * @return returns this object
+		 */
+		public Builder displayWhen(DynamicContent value) {
```

Comment:
I think that a parameter should be of type `Condition` instead of type `DynamicContent`.
That is more in line with the other code. See, for instance, method `public Builder condition(Condition condition) {` in file `VolumeTemplateProperties.java`.
Note also that the API has no-where a method `useWhen` so it seems strange to introduce a method `displayWhen` here. I would expect here also something like `public Builder condition(Condition condition) {`.